lib/src/pal/ref/mli_math.h

```diff
@@ -25,6 +25,16 @@ MLI_FORCE_INLINE T mli_math_limit_fx(T sign) {
     return sign < (T)0 ? std::numeric_limits<T>::lowest() : std::numeric_limits<T>::max();
 }
 
+template <typename T>
+MLI_FORCE_INLINE T mli_math_asr_pos_nbits_fx(T x, int nbits)
+{
+    // This function is internal to this file
+    // and is required to avoid compiler issues.
+    if (nbits > (sizeof(T) * 8 - 1))

```

Comment:
mli_math_asr allows nbits to be negative. I don't see how it is handeled here.

Reply:
This function is only called for positive nbits.